server/sonar-server/src/main/java/org/sonar/server/util/ClassLoaderUtils.java

```diff
@@ -110,7 +102,7 @@ public boolean apply(@Nullable String path) {
         Enumeration<JarEntry> entries = jar.entries();
         while (entries.hasMoreElements()) {
           String name = entries.nextElement().getName();
-          if (name.startsWith(rootDirectory) && predicate.apply(name)) {
+          if (name.startsWith(rootDirectory) && predicate.test(name)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS134)